src/modules/local_position_estimator/sensors/beacon.cpp

```diff
@@ -0,0 +1,127 @@
+#include "../BlockLocalPositionEstimator.hpp"
+#include <systemlib/mavlink_log.h>
+#include <matrix/math.hpp>
+
+extern orb_advert_t mavlink_log_pub;
+
+static const uint64_t 	BEACON_TIMEOUT =   2000000; // [us]
+
+void BlockLocalPositionEstimator::beaconInit()
```

Comment:
Again, we can generalize this to be used as the structure code for a fixed landmark/beacon to aid on position estimator.